staging/src/k8s.io/apimachinery/pkg/apis/meta/v1/types.go

```diff
@@ -528,6 +533,13 @@ type PatchOptions struct {
 	// flag must be unset for non-apply patch requests.
 	// +optional
 	Force *bool `json:"force,omitempty" protobuf:"varint,2,opt,name=force"`
+
+	// FieldManager is the name used to track resources for this
+	// manager. This field is required for apply requests
+	// (application/apply-patch) but forbidden for non-apply patch
+	// types (jsonpatch, mergepatch, smp).
```

Comment:
Why is this forbidden for non apply requests? It isn't forbidden for Update and Create

Reply:
I forgot to update it :-)

Reply:
Same comment about case (we use camelCase).